shard-operator/src/main/java/com/redhat/service/smartevents/shard/operator/BridgeExecutorServiceImpl.java

```diff
@@ -86,6 +86,12 @@ public void createBridgeExecutor(ProcessorDTO processorDTO) {
 
             // create or update the secrets for the bridgeExecutor
             createOrUpdateBridgeExecutorSecret(bridgeExecutor, processorDTO);
+        } else {
```

Comment:
In what case is this branch called? Why do we need this new code here?

Reply:
It _could_ happen if an attempt is made to re-provision a `Processor` without change.

I experienced this exact issue with `Bridge` on this PR (when updating an Error Handler the `Bridge` itself is not changed but our provisioning mechanism passes it through the Shard Operator by virtue of the `Bridge` moving from `ACCEPTED`->`PREPARING` and its dependencies being `READY`).

I'd err on the side of caution and keep this code in the `BridgeExecutorServiceImpl` too. We didn't know it was an issue for `Bridge`'s until this PR to update the Error Handler. It does not harm, but does prevent a potential issue in the future.

Reply:
ok, fine for me